src/core/Clock.js

```diff
@@ -125,10 +127,4 @@ class Clock {
 
 }
 
-function now() {
```

Comment:
This function comes from a time where `performance` wasn't supported everywhere. Instead of keeping the function and moving it to `utils.js`, it's better to remove it and inline `performance.now()` in `Clock` like in `NodeFrame`.

Reply:
Got it, if I understand correctly, I should have corrected it.